hobbyloop-api/src/main/java/hobbyloop/backend/api/applicationservice/center/CenterApplicationService.java

```diff
@@ -20,13 +24,22 @@ public class CenterApplicationService {
 
 	private final CenterService centerService;
 
+	private final UserService userService;
+
+	public void createCenter(String username, CreateCenterRequestDTO request) {
+		User user = userService.getUserByUsername(username);
+		centerService.createCenter(request.getCenterName(), request.getPhoneNumber(), request.getAddress(),
```

Comment:
보통 이렇게 인자가 많을 때는 이런식으로 작성합니다. 참고만해주세요
```suggestion
		centerService.createCenter(
		  request.getCenterName(), 
		  request.getPhoneNumber(), 
		  request.getAddress(),
		  ... 
		  )
```